### ISSUE TITLE
Version 0.0.6 returns wrong result

### ISSUE THREAD

**Issue by dangkhoasdc** [MicahParks/peakdetect#4]
If I use the following example:

```
lag = 1
influence = 0.0
threshold = 1.0

data = {1, 1, 15, 1, 1}
```

Then 0.0.6 returns all 4 points as Positive signal (1, 1, 1, 1), while version 0.0.5 returns correctly (0, 1, 0, 0).

**Comment by MicahParks** [comment]
Thanks. I took a quick look.

At first glance, it looks like a typo on line 165 in this snippet:
https://github.com/MicahParks/peakdetect/blob/ff727a8979109aac7f381f35c439f88512c199f6/peakdetect.go#L156-L165

This would affect all use cases with a lag of 1.

I need to look around more and make sure before pushing and tagging a change.

**Comment by MicahParks** [comment]
@dangkhoasdc would you review the code change in #5?

**Comment by MicahParks** [comment]
@dangkhoasdc thank you for reviewing that. Please use `v0.1.0`.

I've published a module retraction for `v0.0.6`.
